gcc4mbed/Dockerfile

```diff
@@ -0,0 +1,45 @@
+#
+# To build, from base directory:
+#  docker build --tag shimaore/smoothie2-gcc4mbed gcc4mbed
+#
+FROM debian:testing
+MAINTAINER stephane@shimaore.net
+
+RUN \
+  dpkg --add-architecture i386 && \
+  apt-get update && \
+  apt-get install -y --no-install-recommends \
+    ca-certificates \
+    lbzip2 \
+    make \
+    wget \
+    libc6:i386 \
+    libncurses5:i386 \
+    libstdc++6:i386
+
+COPY . /opt/smoothie/gcc4mbed
+WORKDIR /opt/smoothie/gcc4mbed
+
+RUN \
+  yes | ./linux_install && \
+  rm -r \
+    external/mbed/Release \
+    external/mbed/libraries/mbed/targets/cmsis/TARGET_ARM_SSG \
+    external/mbed/libraries/mbed/targets/cmsis/TARGET_Atmel \
+    external/mbed/libraries/mbed/targets/cmsis/TARGET_Freescale \
```

Comment:
I don't know much about Docker so I can't really comment much on the bulk of this commit but I was wondering why these folders need to be removed? I would have thought that they could be left sitting around or are you trying to minimize the size of the container?